python/graphscope/tests/unittest/test_graph.py

```diff
@@ -190,49 +182,6 @@ def test_load_only_from_efile(
     assert sorted(r_out1) == sorted(r_out2)
 
 
-def test_graph_to_numpy(arrow_property_graph):
```

Comment:
Already test in [test_lazy.py](https://github.com/alibaba/GraphScope/blob/main/python/graphscope/tests/unittest/test_lazy.py#L189) with  `lazy` mode